### ISSUE TITLE
Disable Oni errors?

### ISSUE THREAD

**Issue by cj** [onivim/oni#2719]
Oni version 0.3.9.

Is it possible to disable Oni errors completely as they are producing false positives?

![image 2019-02-10 at 10 45 33 am](https://user-images.githubusercontent.com/1819/52536618-8ed56b00-2d22-11e9-9103-dd6be4261196.png)

**Comment by CrossR** [comment]
I don't think we have an explicit way to disable a language server currently, though I have used `"language.python.languageServer.command": "",` in some projects where I've not bothered to set it up properly to disable it.

Its less "disable the errors" and more "lie to Oni and tell it the wrong command to get the errors, so it can't get any", but the result should be the same. Just swap the language to whichever language you need.

**Comment by cj** [comment]
@CrossR Thanks for the response -- I actually forgot to mention I tried doing `"language": {},` and `"language.typescript.textMateGrammar": {}` before with no luck, I just tried `"language.typescript.languageServer.command": ""` and the errors still show...
